lib/quoters/WebhookQuoter.ts

```diff
@@ -241,6 +243,10 @@ export class WebhookQuoter implements Quoter {
       );
 
       //if valid quote, log the opposing side as well
+      // do not await
```

Comment:
why not? just to not block? does this mean we could get a rejection here and not see it?

Reply:
yeah just to have as little latency as possible on the hot path. Even if there are occasional rejects I think that's fine?